docs/examples/notebooks/02-set_neuron_params.ipynb

```diff
@@ -0,0 +1,321 @@
+    "\n",
+    "This is an example of how to set neuron parameters, such as encoder and decoder\n",
+    "matrices, on the FPGA. This is useful if you've presolved weights (encoders and\n",
+    "decoders) elsewhere or if you want to start in a known state.\n",
+    "\n",
+    "The `FpgaPesEnsembleNetwork` encapsulates a Nengo `Ensemble` and `Connection`\n",
+    "object, each of which you can modify. Modifying the attributes of the `Ensemble`\n",
+    "object alters the neuron parameters simulated on the FPGA, while modifying the\n",
+    "attributes of the `Connection` object alters the parameters of the FPGA ensemble\n",
+    "output connection that is simulated on the FPGA.\n",
+    "\n",
+    "In order to change the attributes of the Ensemble or Connection objects, first\n",
+    "create the FpgaPesEnsembleNetwork object:\n",
+    "\n",
+    "    fpga_ens = FpgaPesEnsembleNetwork(...)\n",
+    "\n",
+    "From which you can access the encapsulated `Ensemble` and `Connection` objects\n",
+    "(using the `.ensemble` and `.connection` attributes of the `FpgaPesEnsembleNetwork`\n",
+    "object respectively):\n",
+    "\n",
+    "    fpga_ens.ensemble\n",
+    "    fpga_ens.connection\n",
+    "\n",
+    "Use these two objects to set specific attributes as desired.\n",
+    "Please look at the Nengo documentation for\n",
+    "[Ensembles](https://www.nengo.ai/nengo/frontend_api.html#nengo.Ensemble) and\n",
+    "[Connections](https://www.nengo.ai/nengo/frontend_api.html#nengo.Connection)\n",
+    "for a full list of options.\n",
+    "\n",
+    "## Step 1: Set up Python Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import nengo\n",
+    "from nengo.solvers import NoSolver\n",
+    "\n",
+    "import nengo_fpga\n",
+    "from nengo_fpga.networks import FpgaPesEnsembleNetwork"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Choose an FPGA Device\n",
+    "\n",
+    "Define the FPGA device on which the remote FpgaPesEnsembleNetwork will run.\n",
+    "This name corresponds with the name in your `fpga_config` file. Recall that\n",
+    "in the `fpga_config` file, device names are identified by the square brackets\n",
+    "(e.g., **[de1]** or **[pynq]**). The names defined in your configuration file\n",
+    "might differ from the example below. Here, the device **de1** is being used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "board = 'de1'  # Change this to your desired device name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Create Some Weights\n",
+    "\n",
+    "Generally, the `encoder` and `decoder` matrices can be generated\n",
```

Comment:
Found a better way to word this to avoid the double "this example" (couldn't think of one yesterday):

... computationally functional model. In this notebook, Nengo will be used...